src/scrcmd_party.c

```diff
@@ -659,7 +659,7 @@ BOOL ScrCmd_MonGetRibbonCount(SCRIPTCONTEXT *ctx) {
 
     u16 i, count;
     for (i = 0, count = 0; i < RIBBON_MAX; i++) {
-        if (GetMonData(pokemon, sub_0208E55C(i, 0), NULL)) {
+        if (GetMonData(pokemon, GetRibbonAttr(i, 0), NULL)) {
```

Comment:
```suggestion
        if (GetMonData(pokemon, GetRibbonAttr(i, RIBBONDAT_MONDATNO), NULL)) {
```
(same goes for the other three instances in this file)